core/src/executor/evaluate/function.rs

```diff
@@ -780,6 +780,35 @@ pub fn to_timestamp<'a>(
     }
 }
 
+pub fn add_month<'a>(
+    name: String,
+    expr: Evaluated<'_>,
+    size: Evaluated<'_>,
+) -> Result<Evaluated<'a>> {
+    let size = eval_to_int!(name, size);
+    let expr = chrono::NaiveDate::parse_from_str(&eval_to_str!(name, expr), "%Y-%m-%d").map_err(
+        |err| {
+            let err: EvaluateError = err.into();
+            err
+        },
+    )?;
```

Comment:
nit. it would be good to use `EvaluateError::from`
```rust
let expr = chrono::NaiveDate::parse_from_str(&eval_to_str!(name, expr), "%Y-%m-%d")
    .map_err(EvaluateError::from)?;
```